frontend/apps/marketing/src/contentful/selectors/getImage.ts

```diff
@@ -0,0 +1,19 @@
+import {ExperienceAsset} from '@/types/contentful/ExperienceAsset';
+
+export function getRelativeImageUrl(asset: ExperienceAsset | undefined) {
+  if (asset === undefined) {
+    return undefined;
+  }
+
+  return asset?.fields?.file?.url;
+}
+
+export function getAbsoluteImageUrl(asset: ExperienceAsset | undefined) {
+  if (asset === undefined) {
+    return undefined;
+  }
+
+  const relativeImageUrl = getRelativeImageUrl(asset);
+
+  return relativeImageUrl ? `https:${relativeImageUrl}` : undefined;
+}
```

Comment:
May be simplfied to this: (checked locally, tests still pass successfully)
```suggestion
export function getRelativeImageUrl(asset?: ExperienceAsset) {
  return asset?.fields?.file?.url;
}

export function getAbsoluteImageUrl(asset?: ExperienceAsset) {
  const relativeUrl = getRelativeImageUrl(asset);
  return relativeUrl ? `https:${relativeUrl}` : undefined;
}
```

Reply:
I agree with removing the extraneous `if (asset === undefined)` check. However, the optional parameter is not intended as a parameter must be sent down.